modules/ROOT/pages/faq.adoc

```diff
@@ -233,3 +233,9 @@ systemd:
     - name: docker.service
       mask: true
 ----
+
+== Are Fedora CoreOS x86_64 disk images hybrid BIOS+UEFI bootable?
+
+The x86_64 images we provide can be used for either BIOS (legacy) boot or UEFI boot. They contain a hybrid BIOS/UEFI partition setup that allows them to be used for either. The exception to that is the `metal4k` 4k native image, which is targeted at disks with 4k sectors and https://github.com/coreos/coreos-assembler/blob/12029fea7798fa5d3535eafcf8c3d02f9a6095e4/src/cmd-buildextend-metal#L200-L202[does not have a BIOS boot partition] because 4k native disks are https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/hard-drives-and-partitions#advanced-format-drives[only supported with UEFI].
```

Comment:
Not sure if it's worth linking to cosa code, but meh. Hmm, also wish we had a better link than the Microsoft one, but I can't find one offhand, so WFM! (I found https://lwn.net/Articles/377895/ though that might be too detailed.)

Reply:
Yeah the existing link is fine. I'm sure it'll break at some point and we'll fix it.